project4/3700crawler

```diff
@@ -81,21 +94,56 @@ class WebCrawler:
             except:
                 break
         # Close the socket
-        sock.close()
+        sock.close() 
+        # Return the data
         return data
 
+    def parse_response(self, data):
+        """Returns a dict of the headers and HTML"""
+        response = {}
+        data = data.split(CRLF + CRLF)
```

Comment:
This should be `data.split(CRLF*2, 1)` to only split on the first CRLF*2. This will split up the body too as-is

Reply:
Gotcha.